sormas-rest/src/main/java/de/symeda/sormas/rest/ExternalVisitsResource.java

```diff
@@ -30,6 +35,23 @@ public Boolean isValidPersonUuid(@PathParam("personUuid") String personUuid) {
 		return FacadeProvider.getPersonFacade().isValidPersonUuid(personUuid);
 	}
 
+	@POST
+	@Path("/person/{personUuid}/status")
+	public boolean postSymptomJournalStatus(@PathParam("personUuid") String personUuid, Map<String, Object> body) {
```

Comment:
why not create here a specific transfer object for the body?

Reply:
Conventic gave us [some API specification](https://github.com/hzi-braunschweig/SORMAS-Project/issues/1970) with one field that we don't need (statusDateTime). Instead of putting this into a transfer object, I preferred this more flexible approach. Imagine that, in the future, some other symptom journals want to give us more or other additional information. With this more generic approach, we are better prepared for that. Is that convincing?

Reply:
not really, we should be having one standardised API and not expect everything
nonetheless with a TO this expectation is also doable as it can be extended and at least if the body doesn't correspond to the object is handled by javax rs and no need to catch a possible nullpointer